src/i18n/TextCollection/i18nTextCollector.php

```diff
@@ -511,7 +534,7 @@ protected function getFileListForModule(Module $module)
         $modulePath = $module->getPath();
 
         // Search all .ss files in themes
-        if (stripos($module->getRelativePath() ?? '', 'themes/') === 0) {
+        if (stripos($module->getRelativePath() ?? '', THEMES_DIR . DIRECTORY_SEPARATOR) === 0) {
```

Comment:
No need for this change - the path returned by `$module->getPath()` has already been through `Path::normalise()` which in turn uses `Convert::slashes()` - long story short, at this point we only have unix-style file paths.
This change could actually _stop_ this check from working correctly.

Reply:
done. thanks!